gnss/gps_time.py

```diff
@@ -94,20 +100,21 @@ def gps_minus_utc_seconds(gpst):
       Returns the number (or an array of them) of leap second values.
     """
 
-    delta_utc = np.zeros(gpst.shape, np.int)
+    delta_utc = np.zeros(gpst.shape, int)
     delta_utc = np.array(delta_utc)
-    assert np.all(gpst >= np.datetime64('1999-01-01T00:00:13Z'))
+    assert np.all(gpst >= np.datetime64('1999-01-01T00:00:13'))
     # difference was 16 seconds on 1st July 2012, add the leap seconds since that
     delta_utc += 13
-    delta_utc[gpst >= np.datetime64('2005-01-01T00:00:13Z')] += 1
```

Comment:
I'm guessing there was a reason, but why did the time zone need to get dropped?

Reply:
Recent versions of numpy spit out warnings that time zones are not supported. See https://numpy.org/doc/stable/release/1.11.0-notes.html#datetime64-changes

Reply:
The number of warnings logged was making it hard for me to check the test results